package.json

```diff
@@ -39,4 +39,4 @@
     "url": "http://github.com/ForbesLindesay"
   },
   "license": "MIT"
-}
\ No newline at end of file
+}
```

Comment:
we should bump package json right or else the dependency installer will skip this

Reply:
resolved in person